pom.xml

```diff
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.interledger</groupId>
   <artifactId>java-ilp-core</artifactId>
-  <version>0.5.0-SNAPSHOT</version>
+  <version>0.6.1-SNAPSHOT</version>
```

Comment:
The idea was originally that if you don't use Gradle you check the project out and still build/install it with Maven.